package.json

```diff
@@ -157,6 +157,9 @@
     "minimist": "^1.2.5",
     "elliptic": "^6.5.4",
     "axios": "0.21.1",
-    "redis": "3.1.1"
+    "redis": "3.1.1",
```

Comment:
This redis change is failing on smoke test

Reply:
@srochani  ya I have fixed it in this PR..it's because I didn't commit the lock file... tests passing in this pr